docs/rule/kotlin_test.soy

```diff
@@ -0,0 +1,118 @@
+{namespace kotlin_test}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_test()' /}
+    {param navid: 'rule_kotlin_test' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of kotlin files that contain tests.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
+{param overview}
+A <code>kotlin_test()</code> rule is used to define a set of
+{sp}<code>.kt</code> files that contain tests to run via JUnit.
+{/param}
+
+{param args}
+
+{call buck.arg}
+  {param name: 'name' /}
+  {param desc}
+  The name of the rule.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'srcs' /}
+  {param default : '[]' /}
+  {param desc}
+  Like <a href="kotlin_library.html"><code>kotlin_library()</code></a>,
```

Comment:
Please don't add a link link this.  Add a new entry in `__common.soy` file, and then just replace this link with `{call buck.kotlin_library /}`.